app/assets/stylesheets/modules/style.scss

```diff
@@ -0,0 +1,88 @@
+@import "variables";
+
+body {
+  background-image: image-url("default.jpg");
```

Comment:
Properties should be ordered background-attachment, background-image, background-repeat, background-size, color, font-family, height, position, text-align, width